boards/xtensa/waveshare_esp32s3_touch_lcd/waveshare_esp32s3_touch_lcd-pinctrl.dtsi

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2024 Joel Guittet
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <dt-bindings/pinctrl/esp32s3-pinctrl.h>
+#include <zephyr/dt-bindings/pinctrl/esp-pinctrl-common.h>
+#include <zephyr/dt-bindings/pinctrl/esp32s3-gpio-sigmap.h>
+
+&pinctrl {
+
+	i2c1_default: i2c1_default {
+		group1 {
+			pinmux = <I2C1_SDA_GPIO6>,
+				<I2C1_SCL_GPIO7>;
+			bias-pull-up;
+			drive-push-pull;
+			output-high;
+		};
+	};
+
+	ledc0_default: ledc0_default {
+		group1 {
+			pinmux = <LEDC_CH0_GPIO2>,
+				<LEDC_CH1_GPIO5>,
+				<LEDC_CH2_GPIO4>;
```

Comment:
nit:

```
                        pinmux = <LEDC_CH0_GPIO2>, 
                                 <LEDC_CH1_GPIO5>,
                                 <LEDC_CH2_GPIO4>;
```

(not a hard rule but that's how it's commonly done)

Reply:
This is mixing tab and spaces, right ? So depending of the tab configuration in your editor, sometimes it's aligned and sometimes not (a reason for which I hate tab in source code).

Reply:
By the way, this can fit on 80 characters, so I will remove the \n

Reply:
tabs + space is ok, space + tab or other mixes are not (and should be caught by the presubmit check), it does makes diff odd sometimes, this is just how things are